test/models/list_test.rb

```diff
@@ -47,31 +46,11 @@ class ListTest < ActiveSupport::TestCase
     assert_equal(
       {
         "id"=>980190963,
+        "user_id"=>list.user_id,
         "name"=>"Shopping Place",
-        "items"=>
-        [
-          {
-            "id"=>nil,
-            "name"=>"Item 1",
-            "person"=>"Person 1",
-            "department"=>"Department 1",
-            "bought"=>false,
-            "quantity"=>nil,
-            "active"=>true},
-            {
-              "id"=>nil,
-              "name"=>"Item 2",
-              "person"=>"Person 2",
-              "department"=>"Department 2",
-              "bought"=>false,
-              "quantity"=>nil,
-              "active"=>true
-            }
-          ],
-            :unbought=>0,
-            :item_count=>0,
-            :active=>list.items.active
-        },
+        "created_at"=>list.created_at.strftime('%Y-%m-%dT%H:%M:%S.%LZ'),
+        "updated_at"=>list.updated_at.strftime('%Y-%m-%dT%H:%M:%S.%LZ'),
```

Comment:
This is fine for right now, but if you find you have to hardcode this format string another time, it would be good to extract it into a constant in the test.  If you need to use it in multiple tests, put it in `test/test_helper.rb`.

Reply:
👍🏽